src/test/java/core/basesyntax/strategy/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.operation.BalanceTypeHandler;
+import core.basesyntax.strategy.operation.OperationHandlers;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+class OperationStrategyImplTest {
+    private static Map<FruitTransaction.Operation, OperationHandlers> operationHandlersMap =
+            new HashMap<>() {
+                {
+                    put(FruitTransaction.Operation.BALANCE, new BalanceTypeHandler());
+                }
+            };
+
+    private static final OperationStrategy OPERATION_STRATEGY
+            = new OperationStrategyImpl(operationHandlersMap);
```

Comment:
this should be done in `setUp` method
```suggestion
    private static OperationStrategy strategy;
            
     @BeforeAll
    static void setUp() {
        Map<FruitTransaction.Operation, OperationHandlers> operationHandlerMap = Map.of(FruitTransaction.Operation.BALANCE, new BalanceTypeHandler());

        strategy = new OperationStrategyImpl(operationHandlerMap);
    }
```